src/pipeline/rackhd/docker/Build.groovy

```diff
@@ -0,0 +1,45 @@
+package pipeline.rackhd.docker
+
+def build(String manifest_path, String clone_dir, String workdir){
+    node(build_docker_node){
```

Comment:
Could you please change it to "docker_build"?
We can add label "docker_build" to slaves which are used to build docker images